pipelines/official_build.yml

```diff
@@ -0,0 +1,40 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+schedules:
+- cron: "5 0 * * *"
+  displayName: Daily midnight build
+  branches:
+    include:
+    - master
+  always: true
+
+stages:
+- stage: Build
+  pool: sonicbld
+  jobs:
+    - job:
+      timeoutInMinutes: 720
+      steps:
+      - checkout: self
+        displayName: 'checkout code'
+      - bash: |
+          make init
+          make configure PLATFORM=$(PLATFORM)
+        displayName: 'make init & config'
+      - bash: |
+          make SONIC_CONFIG_BUILD_JOBS=1 INSTALL_DEBUG_TOOLS=y target/sonic-broadcom.bin
+          mv target/sonic-broadcom.bin target/sonic-broadcom-dbg.bin
```

Comment:
please setup the wcache.

Reply:
please refer the pr build, add neccessary comamnd. such as enable pull and trap.

Reply:
changed. so nightly build use wcache, PR build use rcache.
That is the only difference.